src/coalgebras/itreeTauScript.sml

```diff
@@ -898,6 +936,100 @@ Proof
             itree_wbisim_sym]
 QED
 
+Theorem itree_bind_strip_tau_wbisim:
```

Comment:
Maybe also prove the iff result that removes `Tau` constructors?  Something like 
```
wbisim (itree_bind (Tau x) k) y <=> wbisim (itree_bind x k) y
```
And the one with the pattern as the second argument.

Reply:
this lemma strips any finite number of taus, I'm not sure how to generalize it. The statement you suggested can be proven with metis using `itree_bind_thm` and `itree_wbisim_tau` and transitivity

Reply:
I realise it's a trivial consequence, but it's also a great deal more convenient to use for the user as it can apply inside any goal and will be used "for free" by the simplifier with the `simp` annotation.

Reply:
ah I see what you mean and you're right! It wasn't strictly a consequence since `strip_tau` force removes all taus, but it did make the proof of the theorem simpler. Added